### PR TITLE
ci: migrate from codefresh to github actions

### PR DESCRIPTION
**Pull request by tmac1973**
## Summary

These workflows replace the codefresh workflows of the same names as Lacework is migrating off of Codefresh.

## How did you test this change?

These work flows were first tested as much as possible in a non-main branch. Then in the main branch of a single terraform module (terraform-azure-activity-log). 

## Issue
https://lacework.atlassian.net/browse/GROW-2760

### CODE REVIEW

**Comment by badass-aoz** [comment]
Is this ready for review? I see a bunch of test failures here

**Comment by tmac1973** [comment]
Failures were because of creds. I was waiting on approvals to get access to Azure. Think it's fixed now.

**Comment by tmac1973** [comment]
Just waiting on a review.

**Review by badass-aoz — APPROVED** [review]
Thanks for doing the migration!

**Pull request merged**
